arm-compute/2016-03-30/swagger/containerService.json

```diff
@@ -377,7 +409,8 @@
           "properties": {
             "adminUsername": {
               "type": "string",
-              "description": "The administrator username to use for all Linux VMs"
+              "description": "The administrator username to use for all Linux VMs",
+              "pattern": "^[a-z][a-z0-9_-]*$"
```

Comment:
Should username for Linux be restricted to lowercase?  If so, I think the server should add the validation for username, instead of adding validation pattern to the client library.  (Or the validation patterns should be the same both in client library and the server.)  Since the Container service takes upper case usernames without an exception, adding this validation pattern to the client library is not only a breaking change, but also blocking scenarios allowed by the current service.   Can we remove this validation pattern?